### PR TITLE
SCMOD-15154: Updated release notes

### PR DESCRIPTION
**Pull request by kyra-ohare**
https://portal.digitalsafe.net/browse/SCMOD-15154

### CODE REVIEW

File: release-notes-3.5.1.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#### Version Number
+${version-number}
+
+#### New Features
```
Comment (rorytorneymf) [inline release-notes-3.5.1.md:4]:
```suggestion
#### New Features
- None
```

File: release-notes-3.5.1.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+#### Patch Fixes Included" and add
```
Comment (rorytorneymf) [inline release-notes-3.5.1.md]: This doesn't look right, check https://github.com/JobService/worker-batch/releases/tag/v3.2.1 and https://github.com/JobService/worker-batch/releases/tag/v3.2.1

**Review by rorytorneymf — CHANGES_REQUESTED** [review]
If this is a patch release, the build number needs to be changed as it is currently a minor release, You can use the Jenkins re-version repo job to do it:

http://sou-jenkins2.swinfra.net:8080/view/Utility%20Jobs/job/Re-version_Repo/build?delay=0sec

File: release-notes-3.5.1.md [hunk 3]
```diff
@@ -2,8 +2,9 @@
 ${version-number}
 
 #### New Features
+- None
 
-#### Patch Fixes Included" and add
+#### Patch Fixes Included
 - This release includes OS package updates only.
 
 #### Known Issues
```
Comment (rorytorneymf) [inline release-notes-3.5.1.md:10]:
```suggestion
#### Known Issues
- None
```

File: release-notes-3.5.1.md (hunk 2 above)
Comment (kyra-ohare) [reply release-notes-3.5.1.md]: I followed this Michael's[ comment](https://github.com/Autoscaler/autoscaler/pull/69#discussion_r634195792).
Maybe there are two options to approach a patch release!?

**Comment by kyra-ohare** [comment]
I have renamed the file to reflect it is a patch release.

**Comment by rorytorneymf** [comment]
> I have renamed the file to reflect it is a patch release.

No, revert that change, and use the jenkins job to change the version on the repo:

http://sou-jenkins2.swinfra.net:8080/view/Utility%20Jobs/job/Re-version_Repo/build?delay=0sec

**Comment by kyra-ohare** [comment]
Now I see what you mean. Adding these to my personal notes. Thanks.

**Comment by kyra-ohare** [comment]
@rorytorneymf 
How would you like me to proceed with this patch release? Option number [1](https://github.com/JobService/worker-batch/releases/tag/v3.2.1) or [2](https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2)?

**Comment by rorytorneymf** [comment]
> @rorytorneymf How would you like me to proceed with this patch release? Option number [1](https://github.com/JobService/worker-batch/releases/tag/v3.2.1) or [2](https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2)?

I think either is fine, maybe https://github.com/CAFapi/opensuse-base-image/releases/tag/v1.4.2 if that's what @michael-bryson  suggested

**Review by rorytorneymf — APPROVED** [review]

**Pull request merged**